### PR TITLE
logging: replace prints by logging

### PR DESCRIPTION
**Pull request by ppanero**
- closes https://github.com/inveniosoftware/invenio-rdm-migrator/issues/93

### CODE REVIEW

File: invenio_rdm_migrator/streams/runner.py [hunk 1]
```diff
@@ -91,7 +97,6 @@ def __init__(self, stream_definitions, config_filepath):
                             existing_data=existing_data,
                             **stream_config.get("load", {}),
                         ),
-                        logger=self.logger,
```
Comment (ppanero) [inline invenio_rdm_migrator/streams/runner.py:94]: What I found online they avoid drilling the attribute, using the getter instead. In some cases they create factories (i.e. encapsulate the getter, even cache it). I think for now, that is a bit over-engineering it.

File: invenio_rdm_migrator/load/postgresql.py [hunk 2]
```diff
@@ -110,7 +113,7 @@ def _load(self, table_entries):
                     # total file size for progress logging
                     file_size = fpath.stat().st_size
 
-                    print(f"[{ts()}] COPY FROM {fpath}")  # TODO: logging
+                    logger.info(f"[{ts()}] COPY FROM {fpath}")  # TODO: logging
```
Comment (rekt-hard) [inline invenio_rdm_migrator/load/postgresql.py:116]: Can the TODO be removed?

**Review by rekt-hard — COMMENTED** [review]
just a general comment: would it be possible to set the logger in a general place for easy reuse? Or at least specify the logger name in order to not have to explicitly write it every time?

**Comment by anikachurilova** [comment]
LGTM! Just a question: should we cover the logging with tests or it's a waste of time?

**Review by kpsherva — APPROVED** [review]
LGTM, the config of the logger is the responsibility of the instance using it, correct?

**Comment by zzacharo** [comment]
> LGTM, the config of the logger is the responsibility of the instance using it, correct?

yes but is required among others that have changed. We need to change the streams yaml in [cds-rdm](https://github.com/CERNDocumentServer/cds-rdm/issues/71) similar to zenodo-rdm.

**Comment by ppanero** [comment]
@anikachurilova while I am a big supporter of testing everything, I'm not sure if testing the output of the logs is important. I'd agree on doing functional unit testing (i.e. what is supposed to happen, happens). In that sense, we could test that the appropriate csv lines, caches, etc. and the exceptions are thrown when expected. However, doing all those tests is out of scope of this PR.

What do the rest think?

**Comment by alejandromumo** [comment]
> just a general comment: would it be possible to set the logger in a general place for easy reuse? Or at least specify the logger name in order to not have to explicitly write it every time?

We agree on this, c.c. @anikachurilova .

About testing logs, I agree with @ppanero and I don't think we need to add tests for that now.

**Review by anikachurilova — APPROVED** [review]

**Pull request merged**